pkg/agent/agent.go

```diff
@@ -44,21 +51,30 @@ const (
 	ContainerNotExist    = 3
 	CniTryLater          = 11
 	DriverFailure        = 101
+	FastPathFailure      = 102
 	InternalIPAM         = "nimbess"
+	FastPathNetwork      = "fastpath"
+	DefaultBaseCNIDir    = "/var/lib/nimbess/cni"
+	DefaultPodNs         = "default"
 )
 
-// TODO REMOVE this when IPAM is supported from CNI side
+// For internal Nimbess IPAM
```

Comment:
keep TODO here. Also, if we are not planning to call external ipam, then also add TODO to change those IPs to ipv6.

Reply:
will add